cmd/testscript/help.go

```diff
@@ -31,6 +31,11 @@ proxy server. See the documentation for
 github.com/rogpeppe/go-internal/goproxytest for details on the format of these
 files/directories.
 
+Environment variables can be passed through to each script with the -e flag,
+where VAR is the name of the variable. Variables override testscript defined
+values, with the exception of WORK which cannot be be overridden. The -e flag
+can appear multiple times.
```

Comment:
s/can appear multiple times/can appear multiple times to specify multiple variables./

?

Reply:
Thanks.